src/parser/parsers/items.py

```diff
@@ -113,3 +116,6 @@ def _format_pipe_sep_string(self, pipe_sep_string, map_func):
             output_array.append(mapped_value)
 
         return output_array
+    
+    def _is_enabled(self):
+        return not self.get('Disabled', False)
```

Comment:
Better for item logic to happen in item parser class than in the destination

Reply:
This function isn't used anywhere though

Reply:
Its used by PR #41 to not tag disabled items, i.e. "Fire Rate"

Reply:
Sure, so this doesn't need to be here then if its covered by a different PR

Reply:
its more relevant to this PR than to #41 , #49 was merged into #41 so it could be tested before 41 is moved to develop. I presume the confusion is due to 49 being created after 41, but #49 being ready to merge before #41

Reply:
so to clarify; this branch, #49, adds _is_enabled(), and can be merged to develop. #41, the changelog branch, will utilize the _is_enabled() function, but #41 will not be merged for a while.

Reply:
This is just on the principal that any new function should be used by something. In your [other PR it includes this exact change](https://github.com/deadlock-wiki/deadbot/pull/41/files#diff-3cbbd542b0f3226fd50f541a9b17768c720e1ca4c61b7ba039e4d90fcc27d195R120-R121). If this PR doesn't get merged for a while, fine, but just add whatever function you need when the time comes

And even if this function was used, I don't think it does anything. `self.Disabled` doesn't exist does it?

Reply:
I understand this principal but more importantly than this, its makes much more sense for _is_enabled() to be added with the branch that adds the IsDisabled flag, than for it to be added to a PR that has nothing to do with it. If were looking thru history to find out where _is_enabled() was added, we would never think to check a PR named 'changelog-updates', instead, we would check the branch thats named off the function's purpose, 'item disabled flag'. 

If the function was completely unrelated to this PR, I would agree that it can wait to be added with #41.

Reply:
It's quite hard to back this when the function doesn't appear to do anything. Have I missed something?

Reply:
Fixed. Eventually, we should look into instantiating Item objects instead of ItemParser objects.